### PR TITLE
Discord Signup: Implement feedback on text/images

### PR DESCRIPTION
**Pull request by rshipp**
### Description

Add more text/images to discord signup process, based on early feedback.

### Motivation and Context

Tasks:
* https://app.asana.com/0/0/1201339900860375/f 
* https://app.asana.com/0/0/1201339900860379/f 
* https://app.asana.com/0/0/1201339900860376/f 
* https://app.asana.com/0/0/1201339900860377/f

### Screenshots (if appropriate):

Screen Shot 2021-11-17 at 12 17 24 PM
Screen Shot 2021-11-17 at 12 17 36 PM

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/views/discord_signups/launch.html.erb [hunk 1]
```diff
@@ -2,16 +2,21 @@
 <h1>Get Started With Discord</h1>
 
 <% if @discord_user.nil? %>
-  <p>Complete Step 1, then come back to this page and complete the remaining steps.</p>
+  <p>Complete Step 1, then come back to this page and complete the remaining steps. These steps are specific to you and your Discord account, so they may change as you go through the process.</p>
```
Comment (rshipp) [inline app/views/discord_signups/launch.html.erb:5]: @elenabaurkot lmk if this seems ok, or if you had something else in mind

Comment (elenabaurkot) [reply app/views/discord_signups/launch.html.erb:5]: That looks good to me! @rshipp

**Review by elenabaurkot — COMMENTED** [review]
This all looks good to me, I think the updates make everything pretty clear!

**Review by sadleb — APPROVED** [review]
Lgtm

**Pull request merged**